### DISCUSSION TITLE
View IFF image with any features

### DISCUSSION THREAD

**Discussion by Real-MJoe** [prb28/vscode-amiga-assembly#169] [Ideas]
It is possible to spend the "IFF image viewer" any feature? A simple zoom is usefull. Displaying the used color palette will be also great. Or a fact box with infos e.g. size,resolution and so on.

**Comment by prb28** [comment]
Yes, but looks like some work... Could be a good pull request ;-).

**Comment by prb28** [comment]
Please add it to the features request, unless I'll forgot it.
